lambdaclass/madara_explorer
# Block events tab

Implementation of the block events tab.

# Testing the PR:
- Go the [`blocks`](http://localhost:4000/mainnet/blocks) window.
- Select any block by clicking in the `block number`.
- Go to the `Events` tab.

The fields: `Identifier`, `Block Number`, `Transaction Hash` and `From Address` should redirect you to the corresponding show.

# Note

The field `Name` will be added in another PR since it is a considerable amount of work.

closes #114
